packages/core/src/web/app/components/dialogs/DocumentSettings/__snapshots__/index.spec.tsx.snap

```diff
@@ -4521,9 +4521,9 @@ exports[`test DocumentSettings should render correctly for promark 1`] = `
                               </span>
                               <span
                                 class="ant-select-selection-item"
-                                title="High (500 DPI)"
+                                title="Medium (250 DPI)"
```

Comment:
this snapshot is changed because the state is renew to default for each test (medium) in new test.
However the old test would not, so it would use the changed value by previous test (high).